tests/__snapshots__/test_docs_generation_output.ambr

```diff
@@ -0,0 +1,115 @@
+  
+  Syntax
+  ===============================================================================
+  
+  .. code-block:: console
+  
+    snow cortex complete
+      <text>
+      --model <model>
+      --file <file>
+      --connection <connection>
+      --account <account>
+      --user <user>
+      --password <password>
+      --authenticator <authenticator>
+      --private-key-path <private_key_path>
+      --token-file-path <token_file_path>
+      --database <database>
+      --schema <schema>
+      --role <role>
+      --warehouse <warehouse>
+      --temporary-connection
+      --mfa-passcode <mfa_passcode>
+      --enable-diag
+      --diag-log-path <diag_log_path>
+      --diag-allowlist-path <diag_allowlist_path>
+      --format <format>
+      --verbose
+      --debug
+      --silent
+  
+  Arguments
+  ===============================================================================
+  
+  :samp:`{text}`
+    Prompt to be used to generate a completion. Cannot be combined with --file option.
+  
+  Options
+  ===============================================================================
+  
+  :samp:`--model {TEXT}`
+    String specifying the model to be used. Default: snowflake-arctic
+  
+  :samp:`--file {FILE}`
+    JSON file containing conversation history to be used to generate a completion. Cannot be combined with TEXT argument.
+  
+  :samp:`--connection, -c, --environment {TEXT}`
+    Name of the connection, as defined in your ``config.toml``. Default: ``default``.
+  
+  :samp:`--account, --accountname {TEXT}`
+    Name assigned to your Snowflake account. Overrides the value specified for the connection.
+  
+  :samp:`--user, --username {TEXT}`
+    Username to connect to Snowflake. Overrides the value specified for the connection.
+  
+  :samp:`--password {TEXT}`
+    Snowflake password. Overrides the value specified for the connection.
+  
+  :samp:`--authenticator {TEXT}`
+    Snowflake authenticator. Overrides the value specified for the connection.
+  
+  :samp:`--private-key-path {TEXT}`
+    Snowflake private key path. Overrides the value specified for the connection.
+  
+  :samp:`--token-file-path {TEXT}`
+    Path to file with an OAuth token that should be used when connecting to Snowflake.
+  
+  :samp:`--database, --dbname {TEXT}`
+    Database to use. Overrides the value specified for the connection.
+  
+  :samp:`--schema, --schemaname {TEXT}`
+    Database schema to use. Overrides the value specified for the connection.
+  
+  :samp:`--role, --rolename {TEXT}`
+    Role to use. Overrides the value specified for the connection.
+  
+  :samp:`--warehouse {TEXT}`
+    Warehouse to use. Overrides the value specified for the connection.
+  
+  :samp:`--temporary-connection, -x`
+    Uses connection defined with command line parameters, instead of one defined in config. Default: False
```

Comment:
Why sometimes there's ending `.` and sometimes not?

Reply:
I left no-dot after default, but no strong opinion. Changed